client.go

```diff
@@ -0,0 +1,78 @@
+package filetypes
+
+import (
+	"github.com/cloudquery/filetypes/csv"
+	"github.com/cloudquery/filetypes/json"
+)
+
+type FormatType string
+
+const (
+	FormatTypeCSV  = "csv"
+	FormatTypeJSON = "json"
```

Comment:
```suggestion
	FormatTypeCSV  FormatType = "csv"
	FormatTypeJSON FormatType = "json"
```

Reply:
Why not `FormatTypeCSV = FormatType("csv")`? :)